src/dispatch.rs

```diff
@@ -169,10 +171,12 @@ impl<'t> Dispatcher<'t> {
 
         #[cfg(target_os = "emscripten")]
         self.dispatch_seq(res);
+
+        self.dispatch_thread_local(res);
```

Comment:
Okay, so do you like my proposal with two methods? I assumed it would solve your case, because you wrote that you sometimes want to execute the thread-local systems but not always.